Spec/RealtimeClientPresence.swift

```diff
@@ -3590,6 +3590,174 @@ class RealtimeClientPresence: QuickSpec {
                         }
                     }
 
+                    // RTP11c2
+                    fit("should return members filtered by clientId") {
+                        let options = AblyTests.commonAppSetup()
+                        let now = NSDate()
+
+                        var clientMembers: ARTRealtime?
```

Comment:
Is this used for anything? It doesn't seem to, as then you use `presenceData` as mock?

Reply:
@tcard Just to be sure it filters different connectionId's from 101 members.

Reply:
Sorry, I don't follow. Where are you checking that? I only see that you check against the `presenceData` dataset, not against the members you actually push.

Reply:
The query to filter members by `connectionId` https://github.com/ably/ably-ios/blob/fa2cf962304fd1d968813b808b62f483cb046d05/Spec/RealtimeClientPresence.swift#L3743

Reply:
But 1. that's not even in this test, and 2. in that query you're also checking against the `presenceData` dataset by mocking the SYNC process, so `clientMembers` are never retrieved unless I'm missing something.

Reply:
Yes, you're right. I was looking at the other test. I'll change it.

Reply:
Done d157cb2.